lib/llm/message.rb

```diff
@@ -4,9 +4,14 @@ module LLM
   class Message
     attr_accessor :role, :content
 
-    def initialize(role, content)
+    def initialize(role, content, extra = {})
       @role = role
       @content = content
+      @extra = extra
+    end
+
+    def logprobs
```

Comment:
@antaz 
Can we link to the relevant OpenAI docs that document the structure of this response ? And discuss logprobs generally ?

Reply:
@0x1eef Yes! is there a good practice for documenting something like that?

Reply:
we can use @see i guess? in Message#logprobs?

Reply:
Yep, with `@see`